mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/AttachmentMapperTest.java

```diff
@@ -145,7 +147,8 @@ void getAttachmentsShouldReturnEmptyListWhenNonReferencedAttachmentId() {
     }
 
     @Test
-    void getAttachmentsShouldReturnTheAttachmentsWhenSome() throws Exception {
+    @Disabled
```

Comment:
Why disabling it? What about other implems (memory, cassandra) relying on that test?

Reply:
You are absolutely right. I apologize for not noticing this earlier.

Reply:
No worries, that's what code reviews are for :)